operate/archiver/src/main/java/io/camunda/operate/archiver/Archiver.java

```diff
@@ -18,12 +18,10 @@
 import org.springframework.beans.factory.BeanFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Qualifier;
-import org.springframework.context.annotation.DependsOn;
 import org.springframework.scheduling.concurrent.ThreadPoolTaskScheduler;
 import org.springframework.stereotype.Component;
 
 @Component
-@DependsOn("schemaStartup")
 public class Archiver {
```

Comment:
❓ Now, this will `startArchiving` without waiting for `schema startup. Should we keep this, and wait for it to be removed when archiving is completed in exporter?

Reply:
I would propose to remove this. Archiver may fail several times before the indices become available, but it won't die. We will have dirty log on startup but only until we move to archiver.

Reply:
Yes, as soon as we're done migrating the E2E tests we can remove the old archiver.